package.json

```diff
@@ -10,14 +10,21 @@
         "fakeapi": "json-server --watch fakeapi/db.json --routes fakeapi/routes.json --port 9002",
         "start": "npm-run-all --parallel serve fakeapi",
         "lint": "eslint --ext .jsx --ext .js src/",
-        "test": "cross-env NODE_ENV=testing jest \"./src\""
+        "test": "cross-env NODE_ENV=testing jest \"./src\"",
+        "prettier": "prettier \"**/*.js\""
```

Comment:
Do you check this line? Run it locally? Not need prettify only files in `src` directory?

Reply:
Sorry i didn't get it. Btw i've tested prettier script. Do you mean to constrict the scope of prettier to `src` folder!

Reply:
Yep. I think it's unnecessary to check `webpack.dev.js` or other scripts in root folder

Reply:
yeah you are right :+1: